ublox/src/ubx_packets/cfg_val.rs

```diff
@@ -1288,3 +1362,49 @@ pub enum TModePosType {
     /// Lat/Lon/Height position
     LLH = 1,
 }
+
+#[derive(Debug, Default, Clone, Copy, PartialEq)]
+#[cfg_attr(feature = "serde", derive(serde::Serialize, serde::Deserialize))]
+pub enum NavSpgFixMode {
+    /// 2D only
+    Only2D = 1,
+    /// 3D only
+    Only3D = 2,
+    /// Auto 2D/3D
+    #[default]
+    Auto = 3,
+}
+
+#[derive(Debug, Default, Clone, Copy, PartialEq)]
+#[cfg_attr(feature = "serde", derive(serde::Serialize, serde::Deserialize))]
+pub enum NavSpgUtcStandard {
+    /// Automatic; receiver selects based on GNSS configuration
+    #[default]
+    Auto = 0,
+    /// UTC as operated by the U.S. Naval Observatory (USNO); derived from GPS time
+    USNO = 3,
+    /// UTC as operated by the former Soviet Union; derived from GLONASS time
+    SU = 6,
+    /// UTC as operated by the National Time Service Center, China; derived from BeiDou time
+    NTSC = 7,
+}
+
+/// Dynamic platform model
+#[derive(Debug, Default, Copy, Clone, PartialEq, Eq)]
+#[cfg_attr(feature = "serde", derive(serde::Serialize, serde::Deserialize))]
+pub enum NavSpgDynamicModel {
```

Comment:
same here, already exist and remains compatible to the previous Nav5 message,
we should use `CfgNav5DynModel` (possibly renamed to `DynamicModel` to make it less tied to "CfgNav5")

Reply:
It doesn't seem to match. This is from the docs I link to in the PR 
<img width="730" height="261" alt="{68EA27AC-A95F-4B23-BCB3-1002B27255D9}" src="https://github.com/user-attachments/assets/1e061233-3999-4bf3-9ffe-53000f1a9d1f" />

Reply:
Ok I see that the bike is from proto v31

Reply:
I renamed it to `NavDynamicModel` and protected the WristWatch and Bike variants under feature flags, and changed the default to be the `Portable` variant as specified in the docs